docs/en/query_language/system.md

```diff
@@ -85,4 +86,20 @@ Enables background data distribution when inserting data into distributed tables
 SYSTEM START DISTRIBUTED SENDS [db.]<distributed_table_name>
 ```
 
+
+### STOP|START MERGES {#query_language-system-stop-start-merges}
+
+Provides possibility to stop and start background merges for all tables in the MergeTree family:
+
+```sql
+SYSTEM STOP|START MERGES
```

Comment:
we'd better use one syntax description that shows that table name is optional, then say that if it's ommitted the command will affect all tables with MergeTree engine